runhouse/rns/secrets.py

```diff
@@ -519,8 +625,8 @@ def read_secrets(cls, from_env: bool = False, file_path: Optional[str] = None):
                 f"Reading secrets from env is not supported for {cls.PROVIDER_NAME}"
             )
         else:
-            private_key_file = file_path or cls.PRIVATE_KEY_FILE
-            public_key_file = file_path or cls.PUBLIC_KEY_FILE
+            private_key_file = cls.PRIVATE_KEY_FILE
+            public_key_file = cls.PUBLIC_KEY_FILE
```

Comment:
Why do we ignore file_path here?

Reply:
(I see it was wrong before, we need to add ".pub" to it to get the public key file)